bugbug/models/performancebug.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bug_features, bugzilla, feature_cleanup, utils
+from bugbug.model import BugModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class PerformanceBugModel(BugModel):
+    def __init__(self, lemmatization=False):
+        BugModel.__init__(self, lemmatization)
+
+        self.calculate_importance = False
+
+        self.is_performance_bug = bug_features.IsPerformanceBug()
+
+        feature_extractors = [
+            bug_features.HasSTR(),
+            bug_features.Keywords(
+                prefixes_to_ignore=set(self.is_performance_bug.keyword_prefixes)
```

Comment:
We could access `keyword_prefixes` directly on the class since it is a class attribute:
 ```suggestion
                 prefixes_to_ignore=set(bug_features.IsPerformanceBug.keyword_prefixes)
```